tools/src/test/java/org/infinispan/tools/ConfigurationConverterTest.java

```diff
@@ -767,7 +767,7 @@ private void assertPersistenceConverted(EmbeddedCacheManager cm) {
       assertEquals("org.infinispan.persistence.jdbc.configuration.DummyKey2StringMapper", jdbcStringBasedStoreConfiguration.key2StringMapper());
       assertTrue(jdbcStringBasedStoreConfiguration.table().dropOnExit());
       assertTrue(jdbcStringBasedStoreConfiguration.table().createOnStart());
-      assertEquals(50, jdbcStringBasedStoreConfiguration.table().batchSize());
```

Comment:
Yes, the problem was with the Serializer. I've opened https://github.com/infinispan/infinispan/pull/5306 and I'll close this PR